newIDE/app/src/AssetStore/ExtensionStore/ExtensionsSearchDialog.js

```diff
@@ -39,6 +39,7 @@ export default function ExtensionsSearchDialog({
   onCreateNew,
 }: Props) {
   const windowWidth = useResponsiveWindowWidth();
+  const isMobileOrTablet = windowWidth === 'small' || windowWidth === 'xsmall';
```

Comment:
I'm not sure the wording is great: an iPad in landscape is medium I think. But I'm not sure what to use though

Reply:
Correct, an iPad is medium

Reply:
I've updated the logic to only use small now